### PR TITLE
Sort workers by numeric value

### PR DESCRIPTION
**Pull request by sam-github**
connected to strongloop-internal/scrum-nodeops#937

### CODE REVIEW

File: test/test-worker-id-sorting.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// These will be in creation order: always ascending.
+var workerIds = workers.map(function(w) {
+  return String(w.id);
+});
+
+tap.equal(workerIds.length, N, 'should be N workers');
+// Sorting should also be in ascending order.
+tap.strictDeepEqual(liveWorkerIds(), workerIds, 'sort should preserve order');
+
+process.exit(0);
```
Comment (rmg) [inline test/test-worker-id-sorting.js]: This seems like a bad idea to me.. is there something keeping the process alive that makes this necessary?

**Comment by rmg** [comment]
:+1: x :100: for adding a test for this.

**Comment by rmg** [comment]
@slnode test please

File: test/test-worker-id-sorting.js (hunk 1 above)
Comment (sam-github) [inline test/test-worker-id-sorting.js]: All the workers that were just forked keep it alive.

**Comment by sam-github** [comment]
strongloop failing with ENOSPC: http://ci.strongloop.com/job/strongloop/4179/, @rmg

**Comment by rmg** [comment]
Was already looking at the Windows failure.. trying to track down what is using all the disk space. I _think_ I found it, but I'm not sure what to do about it long term yet.

**Comment by sam-github** [comment]
I think this all passed except for 
```
[sqlite3] Success: "C:\jenkins\workspace\strongloop\bdfc0d50\node_modules\minkelite\node_modules\sqlite3\lib\binding\node-v11-win32-x64\node_sqlite3.node" is installed via remote
npm ERR! tar.unpack error reading C:\Users\ADMINI~1\AppData\Local\Temp\tmp4668351618255808570dir\npm-cache\request\2.64.0\package.tgz
npm WARN deprecated showdown@0.3.1: IMPORTANT hotfix
npm WARN locking Error: ENOENT, open 'C:\Users\ADMINI~1\AppData\Local\Temp\tmp4668351618255808570dir\npm-cache\_locks\is-7109025e87abfcee.lock'
npm WARN locking  C:\Users\ADMINI~1\AppData\Local\Temp\tmp4668351618255808570dir\npm-cache\_locks\is-7109025e87abfcee.lock failed { [Error: ENOENT, open 'C:\Users\ADMINI~1\AppData\Local\Temp\tmp4668351618255808570dir\npm-cache\_locks\is-7109025e87abfcee.lock']
npm WARN locking   errno: 34,
npm WARN locking   code: 'ENOENT',
npm WARN locking   path: 'C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp4668351618255808570dir\\npm-cache\\_locks\\is-7109025e87abfcee.lock' }
npm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.1
npm ERR! Attempt to unlock C:\jenkins\workspace\strongloop\bdfc0d50\node_modules\loopback-sdk-angular-cli\node_modules\docular\node_modules\node.extend\node_modules\is, which hasn't been locked
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
npm WARN locking Error: ENOENT, open 'C:\Users\ADMINI~1\AppData\Local\Temp\tmp4668351618255808570dir\npm-cache\_locks\showdown-a6a83a8537ff5a22.lock'
npm WARN locking  C:\Users\ADMINI~1\AppData\Local\Temp\tmp4668351618255808570dir\npm-cache\_locks\showdown-a6a83a8537ff5a22.lock failed { [Error: ENOENT, open 'C:\Users\ADMINI~1\AppData\Local\Temp\tmp4668351618255808570dir\npm-cache\_locks\showdown-a6a83a8537ff5a22.lock']
```

in strongloop, which isn't really effected by this module.

OK to merge?

**Comment by rmg** [comment]
I think it is probably ok.. that failure probably happened while I was blindly deleting everything in an overgrown temp directory.

**Comment by rmg** [comment]
@slnode test please

**Comment by sam-github** [comment]
strongloop build failed on Windows, npm connect errors, caused eslint not to be installed, so pretest failed.

**Pull request merged**